server/src/resolvers/__tests__/authResponse.js

```diff
@@ -0,0 +1,20 @@
+const {AuthResponse} = require('../../resolvers');
```

Comment:
File names must be all lowercase and may include underscores (_) or dashes (-), but no additional punctuation